ledger-tool/src/blockstore.rs

```diff
@@ -39,6 +39,14 @@ use {
     },
 };
 
+/// Blockstore supports two types of access that do not modify the data
+/// (https://github.com/facebook/rocksdb/wiki/Read-only-and-Secondary-instances),
+/// among which Secondary offers extra capabilities that are not used by the ledger tool.
+/// Since there were also cases where secondary access had degraded performance
+/// (https://github.com/anza-xyz/agave/issues/7216), the best is to use ReadOnly when writing
+/// is not needed.
+pub(super) const NON_WRITABLE_BLOCKSTORE_ACCESS: AccessType = AccessType::ReadOnly;
```

Comment:
nit:
- Since we're ripping `Secondary` out, I don't think we need to explicitly mention it. That is, there is no reason to mention `Secondary` for `ledger-tool` if the enum doesn't have a `Secondary` case IMO
- The link to the rocksdb docs is good, but I kind of think it should live with the `AccessType` enum
- `NON WRITABLE` sounds a little awkward to me. Also, since we're ripping out `Secondary` (with no plans to re-add) it, I'm not sure we need this constant at all

Reply:
done